### DISCUSSION TITLE
크롬 콘솔에 스크립트 넣어서 진행중인데 에러가 뜹니다

### DISCUSSION THREAD

**Discussion by mimkim** [kimytsc/covid-rest-vaccine-macro#48] [Q&A] [answered]
보너스 부분에 url 넣고 실행시켰는데,
![image](https://user-images.githubusercontent.com/88809461/129148614-073f0a12-3236-4d84-a815-f89f7fc3351c.png)

에러가 떠서 가보니,
![image](https://user-images.githubusercontent.com/88809461/129148679-ed142df9-1a65-4c74-8705-cbee8203ce6d.png)

여기가 문제인것같은데 어떤부분을 수정해야하나요?

**Comment by kimytsc** [comment, marked as answer]
혹시 예약신청페이지(ex, https://v-search.nid.naver.com/reservation/standby?orgCd=41376633&sid=1085568538) 에 접근하신 뒤에 사용하셨나요?

**Reply by kimytsc** [reply]
네, 그냥 두시면 되며, 예약시도시 `예약시도 상태`에 결과가 노출됩니다.

**Reply by mimkim** [reply]
![image](https://user-images.githubusercontent.com/88809461/129150310-50ff4a12-17b0-4a63-ba0d-c4903b506ba0.png)
감사합니다! 하나만 더 여쭤볼게요, 스크립트에 이슈가 많이 뜨는데 무시해도 되나요?

**Reply by kimytsc** [reply]
병원 리스트를 가져오는데 문제가 발생한 부분인데, 잦은 호출로 인한 일시적인 차단이 아닐까 생각이 듭니다.

혹시 delay를 몇으로 설정하셨나요?

**Reply by mimkim** [reply]
따로 설정하지 않고 제공해주신 스크립트 그대로 사용중입니다 보니까 1000으로 되어있습니다.
![image](https://user-images.githubusercontent.com/88809461/129151410-8ed85775-63c4-4305-9e45-1f17ae7c409e.png)

**Reply by kimytsc** [reply]
DevTools의 Networks에서 빨간색 로그가 지속적으로 나오고 있다면, 일시적으로 차단되었을 가능성이 높고, 한두번씩만 나온다면 문제가 없습니다.

**Reply by mimkim** [reply]
아 그렇군요 근데 현재는 CORS 헤더값을 유효한걸 쓰라는? 뜻으로 보이는 이슈가 뜨는데
![image](https://user-images.githubusercontent.com/88809461/129152445-35d3e223-c255-4f61-ae4d-8f107aaf0f02.png)
저 차단당한건가요? ㅠㅠ

**Reply by mimkim** [reply]
엇 이제 이슈가 안뜹니다 제가 뭔가 잘못했었나봐요 답변들 감사합니다 :)
